src/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs

```diff
@@ -1,206 +1,180 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // See the LICENSE file in the project root for more information. 
 
-//
```

Comment:
I am not sure if we need to remove this author comment because originally the test is picked from external source.

Reply:
@safern may remember the details about that.

---
In reply to: [299625654](https://github.com/dotnet/corefx/pull/38980#discussion_r299625654) [](ancestors = 299625654)

Reply:
Having cleaned it up it isn't the same anymore - everything has basically been rewritten. But probably better to keep it tho to keep the lawyers happy!